docs/src/main/asciidoc/spring-integration-support.adoc

```diff
@@ -253,40 +237,49 @@ class Demo{
 }
 ----
 
+[#receive-from-eh]
 ===== Receive messages from Azure Event Hubs
 Step 1. Fill the credential configuration options.
 
 Step 2. Create a bean of message channel as the input channel.
 [source,java]
 ----
 class Demo{
-    private static final String INPUT_CHANNEL = "input";
-    private static final String EVENTHUB_NAME = "eh1";
-    private static final String CONSUMER_GROUP = "$Default";
-
     @Bean
     public MessageChannel input() {
         return new DirectChannel();
     }
 }
 ----
 
-Step 3. Create `EventHubsInboundChannelAdapter` with the bean of `EventHubsProcessorContainer` to receive messages to Event Hubs.
+Step 3. Create `EventHubsInboundChannelAdapter` with the bean of `EventHubsMessageListenerContainer` to receive messages to Event Hubs.
+
 [source,java]
 ----
-@Bean
+@Configuration
 class Demo{
+    private static final String INPUT_CHANNEL = "input";
+    private static final String EVENTHUB_NAME = "eh1";
+    private static final String CONSUMER_GROUP = "$Default";
+
+    @Bean
     public EventHubsInboundChannelAdapter messageChannelAdapter(
             @Qualifier(INPUT_CHANNEL) MessageChannel inputChannel,
-            EventHubsProcessorContainer processorContainer) {
-        CheckpointConfig config = new CheckpointConfig(CheckpointMode.MANUAL);
-
-        EventHubsInboundChannelAdapter adapter =
-                new EventHubsInboundChannelAdapter(processorContainer, EVENTHUB_NAME,
-                        CONSUMER_GROUP, config);
+            EventHubsMessageListenerContainer listenerContainer) {
+        EventHubsInboundChannelAdapter adapter = new EventHubsInboundChannelAdapter(processorContainer);
         adapter.setOutputChannel(inputChannel);
         return adapter;
     }
+
+    @Bean
+    public EventHubsMessageListenerContainer messageListenerContainer(EventHubsProcessorFactory processorFactory) {
+        EventHubsContainerProperties containerProperties = new EventHubsContainerProperties();
+        containerProperties.setEventHubName(EVENTHUB_NAME);
+        containerProperties.setConsumerGroup(CONSUMER_GROUP);
+        CheckpointConfig config = new CheckpointConfig(CheckpointMode.MANUAL);
```

Comment:
no need for this local variable, we can inline this to `containerProperties.setCheckpointConfig(new CheckpointConfig(CheckpointMode.MANUAL));`